src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -374,7 +374,7 @@ export const MsisdnAuthEntry = React.createClass({
             return (
                 <div>
                     <p>{ _t("A text message has been sent to %(msisdn)s",
-                        { msisdn: <i>this._msisdn</i> },
+                        { msisdn: <i> { this._msisdn }</i> },
```

Comment:
I think that space before the `{` should perhaps go otherwise it'll add inconsistent spacing?

Reply:
@t3chguy you are right. probably I need to delete extra place here. going to do it.